.github/ISSUE_TEMPLATE/enhancement_request.yml

```
@@ -0,0 +1,27 @@
+
+  - type: textarea
+    attributes:
+      label: Solution/Idea
+      description: Describe your solution/idea. If possible, provide mock-up or example of how your idea could look like.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Comments
+      description: Additional information, comments or screenshots about the feature request.
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)